src/pudl/output/ferc1.py

```diff
@@ -3012,15 +3012,55 @@ def out_ferc1__yearly_rate_base(
     data from the nested calculations that are build into the accounting tables.
     See :class:`Exploder` for more details.
 
-    This rate base table also contains one specific addition from
-    :ref:`core_ferc1__yearly_operating_expenses_sched320`. In standard ratemaking
-    processes, utilities are enabled to include working capital - sometimes referred
-    to as cash on hand or cash reverves. A standard ratemaking process is to consider
-    the available rate-baseable working capital to be one eigth of the average
-    operations and maintenance expense. This function grabs that expense and
-    concatenates it with the rest of the assets and liabilities from the granular
-    exploded data.
+    This rate base table also contains one new "cash_working_capital" xbrl_factoid
+    from :ref:`core_ferc1__yearly_operating_expenses_sched320` via
+    :func:`prep_cash_working_capital`.
 
+    We also break down records that have nulls or totals in two of the key tag
+    columns: ``tags_aggregatable_utility_type`` and ``tags_in_rate_base`` via
```

Comment:
I don't totally understand what you mean by "break down" here

Reply:
so fair!.. i think the correct word here is actually "disaggregate". We have records with totals and/or nulls for these key tags which we want to separate those records into component parts.